compute/hybrid/compute.go

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) Microsoft and contributors.  All rights reserved.
+//
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+package hybridcompute
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"github.com/Azure-Samples/azure-sdk-for-go-samples/helpers"
+	"github.com/Azure-Samples/azure-sdk-for-go-samples/iam"
+	hybridnetwork "github.com/Azure-Samples/azure-sdk-for-go-samples/network/hybrid"
+	"github.com/Azure/azure-sdk-for-go/profiles/2017-03-09/compute/mgmt/compute"
+	"github.com/Azure/go-autorest/autorest"
+	"github.com/Azure/go-autorest/autorest/azure"
+	"github.com/Azure/go-autorest/autorest/to"
+)
+
+const (
+	publisher   = "Canonical"
+	offer       = "UbuntuServer"
+	sku         = "16.04-LTS"
+	errorPrefix = "Cannot create VM, reason: %v"
+)
+
+func getVMClient(activeDirectoryEndpoint, tokenAudience string) compute.VirtualMachinesClient {
+	token, err := iam.GetResourceManagementTokenHybrid(activeDirectoryEndpoint, tokenAudience, helpers.TenantID(), helpers.ClientID(), helpers.ClientSecret())
```

Comment:
I would like this function to have less parameters. There is already implemented stuff in the non-hybrid side of the samples. I would like this func to only show the true differences with hybrid.

Reply:
Removed unnecessary parameters